README.md

```diff
@@ -17,6 +17,21 @@ paths=""
 services=""
 </pre>
 
+### proxy_idp_auth_test.sh
+* Attributes to be filled:
+<pre>
+# The url of tested SP
+# For example :https://aai-playground.ics.muni.cz/simplesaml/nagios_check.php?proxy_idp=cesnet
+testSite=""
+
+loginSite="https://idp2.ics.muni.cz/idp/Authn/UserPassword"
```

Comment:
Also use idp2.ics.muni.cz as an example and provide a line with description what is the loginSite variable.

Reply:
Done